build/storage/recipes/scale-out.md

```diff
@@ -0,0 +1,100 @@
+# Scale-out recipe
+
+This recipe describes how to attach 64 virtio-blk disks to a Host
```

Comment:
do we capitalize 'host'?

Reply:
no, it is made lower case in all recipes.